### PR TITLE
Fix changelog format to align with http://keepachangelog.com/en/1.0.0/

### PR DESCRIPTION
**Pull request by nietaki**
The current format is not according to the example on the website, but more importantly has the Added/Changed/Removed sections on the same header level as the releases, making it confusing both in the editor and on GitHub.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1.1](https://github.com/CrowdHailer/Ace/tree/0.1.1) - 2016-10-03
 
-### Fixed
+#### Fixed
```
Comment (CrowdHailer) [inline CHANGELOG.md]: didn't need one here

Comment (CrowdHailer) [reply CHANGELOG.md]: I had this one covered.

Comment (nietaki) [reply CHANGELOG.md]: damn my non-robust regex!

**Comment by CrowdHailer** [comment]
Awesome :taco:

File: CHANGELOG.md (hunk 1 above)
Comment (CrowdHailer) [reply CHANGELOG.md]: `^`

**Pull request merged**